packages/smooth_app/lib/themes/smooth_theme.dart

```diff
@@ -56,11 +56,18 @@ class SmoothThemes {
       brightness: Brightness.light,
     );
 
-    ColorScheme myColorScheme = isDarkTheme ? colorDark : colorLight;
+    final ColorScheme myColorScheme = isDarkTheme ? colorDark : colorLight;
 
     return ThemeData(
       colorScheme: myColorScheme,
-      buttonColor: isDarkTheme ? const Color(0xFF696464) : Colors.black,
+      buttonTheme: ButtonThemeData(
+        textTheme: ButtonTextTheme.accent,
+        colorScheme: Theme.of(context).colorScheme.copyWith(
+              primary: isDarkTheme ? const Color(0xFF696464) : Colors.black,
+              // secondary will be the textColor, when the textTheme is set to accent
```

Comment:
I don't think secondary it the text color.
It's rather an alternative important color, like my whole app is blue (primary) and some action button are orange (secondary). AFAIK.

Reply:
I found it very confusing myself, but it works. 
If you set textTheme to accent you can use colorScheme, and from there secondary is for the text. 
[CLICK ME](https://stackoverflow.com/a/61585500/13313941)

Reply:
Please have a look at it again, as you say it's very confusing.
What if you don't specify anything specific about buttons, but just let the system use the primary and secondary values of your ColorScheme?